wasmer/import.go

```diff
@@ -14,6 +14,139 @@ type ImportedFunctionError struct {
 	message      string
 }
 
+// ImportObjectError represents errors related to `ImportObject`s
+type ImportObjectError struct {
+	message string
+}
+
+// A type that owns a set of imports.
+// It can be combined with a `Module` to create an `Instance`
+type ImportObject struct {
+	inner *cWasmerImportObjectT
+}
+
+// An entry that can be passed to `NewWasiImportObject`.
+// Preopens a file for the WASI module but renames it to the given name
+type MapDirEntry struct {
+	alias string
+	hostPath string
+}
+
+// NewDefaultWasiImportObject constructs a new `ImportObject`
+// with WASI host imports.
+// To specify WASI program arguments, environment variables,
+// preopened directories, and more, see `NewWasiImportObject`
+func NewDefaultWasiImportObject() *ImportObject {
+	var inner = cNewWasmerDefaultWasiImportObject()
+
+	return &ImportObject{inner}
+}
+
+
+// Creates an `ImportObject` with the default WASI imports.
+// Specify arguments (the first is the program name),
+// environment variables ("envvar=value"), preopened directories
+// (host file paths), and mapped directories (host file paths with an
+// alias, see `MapDirEntry`)
+func NewWasiImportObject(args []string, envs []string,
```

Comment:
I think might be a good idea to move the WASI specific stuff to a different file: `wasi.go`